src/cow-react/api/gnosisProtocol/api.ts

```diff
@@ -320,10 +322,9 @@ async function _handleOrderResponse<T = any, P extends UnsignedOrder = UnsignedO
     // Handle response
     if (!response.ok) {
       // Raise an exception
-      const [errorObject, description] = await Promise.all<[Promise<ApiErrorObject>, Promise<string>]>([
-        response.json(),
-        OperatorError.getErrorFromStatusCode(response, 'create'),
-      ])
+      const errorObject: ApiErrorObject = await response.json()
```

Comment:
In this case, we always got an unrecognized error because we tried to read the stream in parallel.
But I got your message, will fix it after the release